pkg/apis/eventbus/v1alpha1/types.go

```diff
@@ -63,9 +63,10 @@ var (
 // NativeStrategy indicates to install a native NATS service
 type NativeStrategy struct {
 	// Size is the NATS StatefulSet size
-	Replicas     int32         `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
-	Auth         *AuthStrategy `json:"auth,omitempty" protobuf:"bytes,2,opt,name=auth,casttype=AuthStrategy"`
-	AntiAffinity bool          `json:"antiAffinity,omitempty" protobuf:"varint,3,opt,name=antiAffinity"`
+	Replicas int32         `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
+	Auth     *AuthStrategy `json:"auth,omitempty" protobuf:"bytes,2,opt,name=auth,casttype=AuthStrategy"`
+	// Deprecated, use Affinity instead, will be removed in v1.5
```

Comment:
Deprecated: xxx

Reply:
Changing to `Deprecated:` will cause lint failure at the place this field being referenced - it does not allow anybody to reference a deprecated property.